src/locales/en.js

```diff
@@ -418,7 +417,6 @@ export default {
     "saintHedwigOfPoland": "Saint Hedwig of Poland",
     "saintHedwigOfSilesia": "Saint Hedwig of Silesia",
     "saintHedwigReligiousOrSaintMargaretMaryAlacoqueVirgin": "Saint Hedwig, Religious Or Saint Margaret Mary Alacoque, Virgin",
-    "saintHedwigReligiousSaintMargaretMaryAlacoqueVirgin": "Saint Hedwig, Religious/Saint Margaret Mary Alacoque, Virgin",
```

Comment:
Is it an `or` or an `and` now? I'm a bit confused.

Reply:
Here I have removed the duplicate key `saintHedwigReligiousSaintMargaretMaryAlacoqueVirgin` and kept the `saintHedwigReligiousOrSaintMargaretMaryAlacoqueVirgin` key only.

And to answer your question explicitly: it is `or`, as there should be two celebrations. If you answer [my previous comment]() and provide me a way to define two celebrations for one day, I’ll do it that way, otherwise this should be okay (according to the old romcal rules).

Reply:
Good 😉